backend/mora/service/address_handler/dar.py

```diff
@@ -97,23 +100,28 @@ def _fetch_from_dar(addrid):
             'adresser', 'adgangsadresser',
             'historik/adresser', 'historik/adgangsadresser'
         ):
-            r = session.get(
-                'https://dawa.aws.dk/' + addrtype,
-                # use a list to work around unordered dicts in Python < 3.6
-                params=[
-                    ('id', addrid),
-                    ('noformat', '1'),
-                    ('struktur', 'mini'),
-                ],
-            )
-
-            addrobjs = r.json()
-
-            r.raise_for_status()
-
-            if addrobjs:
-                # found, escape loop!
-                break
+            try:
+                r = session.get(
+                    'https://dawa.aws.dk/' + addrtype,
+                    # use a list to work around unordered dicts in Python < 3.6
+                    params=[
+                        ('id', addrid),
+                        ('noformat', '1'),
+                        ('struktur', 'mini'),
+                    ],
+                )
+
+                addrobjs = r.json()
+
+                r.raise_for_status()
+
+                if addrobjs:
+                    # found, escape loop!
+                    break
+            # The request mocking library throws a pretty generic exception
+            # catch and rethrow as something we know how to manage
+            except Exception as e:
+                raise LookupError(e)
```

Comment:
The arguments to this exception should be something slightly more helpful; either the arguments of the caller, or a generic stringification of the message. Passing an exception to an exception is… unusual…

Reply:
It gives the exact same result as before, we just handle it earlier, so it seems like it works okay.